spring-cloud-gcp-docs/src/main/asciidoc/datastore.adoc

```diff
@@ -210,7 +210,62 @@ The `@Field` annotation optionally provides a different field name than that of
 
 ==== Embedded Objects
 
-This feature is not supported yet.
+Fields marked as `@Embedded` are converted to Datastore entities and stored inside the parent entity.
```

Comment:
Great work! 

I think it's worth mentioning in the doc that the Embedded annotation here means something different than what it means for SQL style Spring Data modules: here it means it is a sub-struct/sub-row, but for SQL modules it just means using  the columns of the embedded object as columns of the parent. I think it would be good to prevent confusion.